classification1.Rmd

```diff
@@ -709,15 +805,23 @@ For example:
 
 You can find [a full set of all the steps and variable selection functions](https://tidymodels.github.io/recipes/reference/index.html)
 on the recipes home page.
-We finally use the `bake` function to apply the recipe to the unscaled data.
+
+Here we have calculated the required statistics based on the data input into the 
+recipe, but the data are not yet scaled and centred. To actually scale and center 
+the data, we need to apply the bake function to the unscaled data.
+
 ```{r 05-scaling-4}
 scaled_cancer <- bake(uc_recipe, unscaled_cancer)
 scaled_cancer
 ```
 
+It may seem redundant that we had to both `bake` *and* `prep` to scale and center the data.
+However, we do this in two steps so we could specify a different data set in the `bake` step
+if desired, say, new data you want to predict, which were not part of the training set. 
+
 At this point, you may wonder why we are doing so much work just to center and
```

Comment:
"At this point, you may..." -> "You may..."

(since we added "at this point" above, remove it here)